### PR TITLE
V93 koecher candi

### PR DESCRIPTION
**Pull request by koecher**
here is the initial version of my contribution to the release paper.

I've also updated the Acknowledgment section (removed myself and added Fabian Castelli (gfcas github user) due to his mayor contributions to candi in the recent weeks).

Don't hesitate to give me comments for changes or spelling corrections.

### CODE REVIEW

**Comment by tjhei** [comment]
This is a good start, thank you!

Could you start with the explanation that it downloads and compiles deal.ii and it's dependencies from source and runs on various Linux systems and osx. It is especially useful on clusters or other machines without requiring root access to install packages assuming an MPI and c++ compiler is available.

**Comment by koecher** [comment]
I've included the comments from @tjhei and made some other changes without squashing the commits.

Let me know if I should squash my commits.

File: 9.3/paper.tex [hunk 1]
```diff
@@ -918,10 +918,99 @@ \subsection{Evaluation and integration at arbitrary points}
 
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
-\subsection{Terminal based toolchain installer}
+\subsection{Source based toolchain installer}
 \label{subsec:candi}
 
-Half a page up to a page on candi.
+The requirement to download, compile and install \dealii{} and its dependencies from
+source is of mayor interest for many \dealii{}-users.
```
Comment (tjhei) [inline 9.3/paper.tex:925]:
```suggestion
source is of major interest for many \dealii{}-users.
```

File: 9.3/paper.tex [hunk 2]
```diff
@@ -918,10 +918,99 @@ \subsection{Evaluation and integration at arbitrary points}
...
+%
+This can be of importance on arbitrary Linux-based operating systems for which
```
Comment (tjhei) [inline 9.3/paper.tex:927]:
```suggestion
Compiling dependencies from source can be of importance on arbitrary Linux-based operating systems for which
```

File: 9.3/paper.tex [hunk 3]
```diff
@@ -918,10 +918,99 @@ \subsection{Evaluation and integration at arbitrary points}
...
+binary-install package aren't available or on compute clusters and other machines
```
Comment (tjhei) [inline 9.3/paper.tex:928]:
```suggestion
binary packages aren't available or on compute clusters and other machines
```

File: 9.3/paper.tex [hunk 4]
```diff
@@ -918,10 +918,99 @@ \subsection{Evaluation and integration at arbitrary points}
...
+without root privileges for the user to install system dependencies.
+
+The source based installation of \dealii{} or complex toolchains of multiple libraries
```
Comment (tjhei) [inline 9.3/paper.tex:931]:
```suggestion
The source based installation of \dealii{} and it's many dependent libraries
```

File: 9.3/paper.tex [hunk 5]
```diff
@@ -918,10 +918,99 @@ \subsection{Evaluation and integration at arbitrary points}
...
+working together with \dealii{} can be done with the \candi{} script tool for various
```
Comment (tjhei) [inline 9.3/paper.tex:932]:
```suggestion
can be done with the \candi{} script tool for various
```

File: 9.3/paper.tex [hunk 6]
```diff
@@ -918,10 +918,99 @@ \subsection{Evaluation and integration at arbitrary points}
...
+available.
+
+\candi is a bash-script based tool, it is the abbreviation of ``compile and install''
+and it is released under the GNU LGPL v3.0.
+%
+The origin of \candi{} is a fork made in 2013 from \texttt{dorsal},
+a retired source based installer for the \texttt{FEniCS} library.
+%
+The \candi{} tool can be found via the download page of \dealii{} since version 8.5
+and is under active maintenance.
```
Comment (tjhei) [inline 9.3/paper.tex:946]:
```suggestion
or from github.com/dealii/candi and is under active development.
```

**Review by tjhei — COMMENTED** [review]
Looks pretty good. We might want to shorten a few details at the end, but maybe this is good for now.

**Comment by koecher** [comment]
@tjhei Thank you for your comments, I'll handle this in the beginning of next week.

File: 9.3/paper.tex (hunk 4 above)
Comment (bangerth) [reply 9.3/paper.tex:931]: It should be "its", not "it's". I'll fix that myself.

**Comment by bangerth** [comment]
Let's get this merged rather than go back and forth in the PR for a long time. We can always edit the current version if more changes are necessary.

**Comment by bangerth** [comment]
I edited (and shortened slightly) the candi section and it looks good. Thanks for the addition!

**Comment by koecher** [comment]
@bangerth Thank you Wolfgang for the update. If there is something to go over again, let me know.

**Pull request merged**